src/main/java/org/dataone/bookkeeper/resources/UsagesResource.java

```diff
@@ -0,0 +1,342 @@
+     * @return usage the usage object for the given instance identifier
+     */
+    @Timed
+    @GET
+    @PermitAll
+    public List<Usage> listUsages(@Context SecurityContext context,
+                                 @QueryParam("start") @DefaultValue("0") Integer start,
+                                 @QueryParam("count") @DefaultValue("1000") Integer count,
+                                 @QueryParam("quotaType") String quotaType,
+                                 @QueryParam("instanceId") String instanceId,
+                                 @QueryParam("subscribers") Set<String> subscribers,
+                                 @QueryParam("requestor") String requestor) {
+
+        List<Usage> usages = null;
+        Usage usage = null;
+        // The calling user injected in the security context via authentication
+        Customer caller = (Customer) context.getUserPrincipal();
+        boolean isAdmin = this.dataoneAuthHelper.isAdmin(caller.getSubject());
+        Set<String> associatedSubjects;
+        List<String> subjects = new ArrayList<>();
+        Boolean isProxy = isAdmin && requestor != null;
+
+        // Admin users can make request as another user
+        if(requestor != null) {
+            if(isAdmin) {
+                caller.setSubject(requestor);
+            } else {
+                throw new WebApplicationException(caller.getSubject() + " does not have admin privilege needed to set 'requestor'. ", Response.Status.FORBIDDEN);
+            }
+        }
+
+        try {
+            if (subscribers != null && subscribers.size() > 0) {
+                // Filter out non-associated subscribers if not an admin, or if the admin user has requested a proxy requestor
+                if (! isAdmin || isProxy) {
+                    associatedSubjects = this.dataoneAuthHelper.getAssociatedSubjects(caller, subscribers);
+                    if ( associatedSubjects.size() > 0 ) {
+                        subjects.addAll(associatedSubjects);
+                    }
+
+                    /* Caller is not admin and is not associated with any of the specified subscribers. */
+                    if (subjects.size() == 0) {
+                        throw new WebApplicationException(caller.getSubject() + " is not authorized.", Response.Status.FORBIDDEN);
+                    }
+                } else {
+                    /* Admin caller, so can see quotas for all requested subscribers */
+                    subjects.addAll(subscribers);
+                }
+            } else {
+                /* No subscribers specified and caller is not admin, so caller is only allowed to
+                   view their own quotas. If the caller is admin, then don't set subject, as
+                   they will be able to view all subjects. */
+                if (! isAdmin || isProxy) {
+                    if (subjects.size() == 0) {
+                        subjects.add(caller.getSubject());
```

Comment:
@gothub and I discussed expanding the subjects in this list later with associated subjects from the DataONE accounts API using `DataONEAuthHelper`.